docs/dictionary/function/androidRequestPermission.lcdoc

```diff
@@ -0,0 +1,44 @@
+Name: androidRequestPermission
+
+Type: function
+
+Syntax: androidRequestPermission(<permissionName>)
+
+Summary:
+Returns if permission <permissionName> has been granted by the user. If the user has not been asked before, a dialog is displayed,
+showing a permission request for <permissionName>.
+
+Introduced: 9.0.1
+
+OS: mobile
+
+Platforms: android
+
+Example:
+local tCameraPermissionGranted
+put androidRequestPermission("android.permission.CAMERA") into tCameraPermissionGranted
+if not tCameraPermissionGranted then
+   answer "This app is not permitted to access the device camera. You can change this \
```

Comment:
@livecodepanos this does not compile. Should be:
```
answer "This app is not permitted to access the device camera. You can change this" & \
       "in the Settings app."
```
Interestingly enough I chatted to @runrevmark once about `\` escaping newlines in string literals and it's both feasible and backwards compatible to do at some point.

Reply:
Good catch @montegoulding :)

I guess I need to use `&&` instead of `&` (or add a space at the end of the first string)